docker/ubuntu-14.04-cpu-all-options/Dockerfile

```diff
@@ -0,0 +1,72 @@
+FROM ubuntu:14.04
+MAINTAINER Aaron Markham <aaronmarkham@fb.com>
```

Comment:
`MAINTAINER` is deprecated, you should use a label instead:
https://docs.docker.com/engine/reference/builder/#maintainer-deprecated